Resources/Prototypes/_Goobstation/Entities/Clothing/Hands/modsuit.yml

```
@@ -296,3 +297,32 @@
     visualLayers:
       gloves:
       - state: equipped-HAND-sealed
+
+- type: entity
+  parent: [ClothingModsuitGauntletsStandard]
+  id: ClothingModsuitGauntletsBloodredModsuit
+  name: CSA-99MS "Deimos" Modsuit Gauntlets
+  description: A set of armored gauntlets for the "Deimos" modsuit.
+  categories: [HideSpawnMenu]
+  components:
+    - type: Sprite
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)